parse.nix

```
@@ -1,7 +1,7 @@
-{ lib }:
+{ pkgs, lib }:
 
 let
-  inherit (import ./repos/fromElisp {}) fromElisp;
+  inherit (import ./repos/fromElisp {inherit pkgs;}) fromElisp;
```

I would suggest adding the following spacing, which is more in line with de facto nix formatting:
```suggestion
  inherit (import ./repos/fromElisp { inherit pkgs; }) fromElisp;
```